examples/AspNetCore/ControllerSample/Controllers/SampleController.cs

```diff
@@ -58,6 +59,7 @@ public ActionResult<Account> Get([FromState(StoreName)] StateEntry<Account> acco
         ///  "pubsub", the first parameter into the Topic attribute, is name of the default pub/sub configured by the Dapr CLI.
         [Topic("pubsub", "deposit")]
         [HttpPost("deposit")]
+        [Authorize("Dapr")]
```

Comment:
I'd prefer not to change this sample and just add to docs if it's going to make running the samples harder.

Reply:
Removed.